MAUI/Autocomplete/Getting-Started.md

```diff
@@ -7,19 +7,34 @@ control: SfAutocomplete
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI Autocomplete (SfAutocomplete)
+# Getting Started with .NET MAUI Autocomplete
 
-This section explains how to add the Autocomplete control and bind data to it. This section covers only the basic features needed to get started with the Syncfusion [Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) control.
+This section guides you through setting up and configuring a [Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) in your .NET MAUI application. Follow the steps below to add a basic [Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) to your project.
```

Comment:
Remove the API refernence added in the second place

Reply:
Removed.